templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
```

Comment:
We don't include a "v" property (as exists in the "core" ping) because this is a modern "structure ingestion" ping (the namespace is not telemetry). For structured ingestion, we capture the schema version as one of the URI parameters, so we don't need to include it in the body of the message.